src/components/Beers/BeerDetails.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import Navbar from '../Navbar/Navbar';
+import axios from 'axios';
+
+const BeerDetails = (props) => {
+  const [beerDetails, setBeerDetails] = useState({});
+
+  useEffect(() => {
+    const { id } = props.match.params;
+    axios.get(`https://ih-beers-api2.herokuapp.com/beers/${id}`).then((res) => {
+      const beerDetails = res.data;
+      setBeerDetails(beerDetails);
+    });
+  },);
```

Comment:
You should have a dependency array here as well.
```
}, []);

Reply:
It was there, but giving an error on the terminal although compiling the page well, thus I removed it. I can add again.